packages/core/src/components/dropdown-menu/dropdown-menu.scss

```diff
@@ -7,3 +7,17 @@
 :host {
   @include dropdown-menu.root();
 }
+
+.dropdown-menu-container {
+  display: block;
+  visibility: visible;
+  margin: 0 !important;
+  border: 0;
+  padding: 0;
+  background-color: transparent;
+  overflow: unset;
+
+  &[hidden] {
+    visibility: hidden;
+  }
+}
```

Comment:
Dit is niet de eerste keer dat deze styling wordt geintroduceerd om een element met `popover` attribuut te tonen. Maar deze styling is overbodig zodra er `HTMLElement.showPopover()` wordt gebruikt.

En als kers op de kaart komt dit element dan ook nog in de top layer terecht.

Zie hoe het bij Onboarding Tip werkt